deploy/ml-config.xml

```diff
@@ -242,15 +242,15 @@
         <!--
         Sample for:
           <name xmlns="http://marklogic.com/ns/sample">bob<name>
-
+-->
         <range-element-index>
           <scalar-type>string</scalar-type>
-          <namespace-uri>http://marklogic.com/ns/sample</namespace-uri>
-          <localname>name</localname>
+          <namespace-uri></namespace-uri>
+          <localname>region</localname>
           <collation>http://marklogic.com/collation/codepoint</collation>
-          <range-value-positions>false</range-value-positions>
+          <range-value-positions>true</range-value-positions>
```

Comment:
Do you need positions turned on? This is useful for doing near-queries inside of a document.